proptest/src/test_runner/runner.rs

```diff
@@ -548,6 +562,10 @@ impl TestRunner {
             |_| panic!("Ran past the end of the replay"),
             replay.steps.into_iter(),
             ForkOutput::empty(),
+            // Since all of our test runs happened in forks already, we need to
+            // force the runner to count successful replays against our total
+            // success count.
+            true,
```

Comment:
not a huge fan of having boolean flags flip behavior like this. having explicit variants for `PersistedCaseSuccess` and `ReplaySuccess` would remove the need for this right? I think we always want replays to count against the success count and we never want persisted cases to count against success count.

Reply:
updated